src/lsp/cobol_preproc/src_overlay.ml

```diff
@@ -172,5 +173,5 @@ module New_manager (Id: sig val name: string end) () : MANAGER = struct
   let link_limits = link_limits ctx
   let join_limits = join_limits ctx
   let dummy_limit = Lexing.dummy_pos
-  let restart_at = restart_at ctx
+  let restart ?at () = restart ?at ctx
```

Comment:
`at` is not used by `restart`, so what's the point of giving it ?

Reply:
Cheap future-proofing.
It could be quite possible to optimize things in terms of cache management and representation of the limits. The idea is to have an interface that is not likely to change whenever this optimisation is implemented.

Reply:
Given that this optimisation is not going to be done very soon, it is better to keep a non misleading type (i.e. where all arguments are actually used). 

When the optimization will be implemented, it will be easy to change the interface to an extra non-optional argument to find all occurrences that need to be modified.